test/basic-querying.test.js

```diff
@@ -469,6 +469,58 @@ describe('basic-querying', function() {
       });
     });
 
+    it('should support "like" that is satisfied', function(done) {
+      User.find({where: {name: {'like': 'John'}}}, function(err, users) {
+        should.not.exist(err);
```

Comment:
If you decide to go and refactor that file, you can rename the set of tests in this PR too -- the wording is pretty bad -- `it('supports the "like" operator', function(...)` or something along those lines would be better.